layers/+lang/javascript/packages.el

```diff
@@ -260,6 +261,38 @@
           "sR" 'spacemacs/skewer-eval-region-and-focus
           "ss" 'skewer-repl)))))
 
+(defun javascript/init-indium ()
+  (when (eq javascript-repl 'indium)
+    (use-package indium
+      :defer t
+      :init
+      (progn
+        (require 'indium)
```

Comment:
You should not require a package manually, instead this should be done by use-package and the autoload mechanism in emacs.